guide/docs/interactions/buttons.mdx

```diff
@@ -88,13 +56,203 @@ async def help_listener(inter: disnake.MessageInteraction):
 <br />
 
 :::note
-
 `Link` buttons _cannot_ have a `custom_id`, and _do not_ send an interaction event when clicked.
+:::
 
+### Example of Buttons
+
+Button components, like [SelectMenu](select-menus.mdx) components, can be implemented using a <DocsLink reference="disnake.ui.View">View</DocsLink>
+or via a [low-level implementation](#Views-vs-low-level-components).
+
+:::note
+A message can include up to 25 button components, organized with a maximum of 5 buttons per row across 5 rows.
 :::
 
-### Disabled buttons
+```python title="button_view.py"
+# At top of file.
+import disnake
+from disnake.ext import commands
+
+
+class ButtonView(disnake.ui.View):
+    def __init__(self):
+        # Views by default have a 180s `timeout`
+        super().__init__()
+
+    # In this example, the buttons are created using `@disnake.ui.button` (Notice the lower `b`)
+    # to create the buttons and assign the decorated functions as the button callback.
+
+    @disnake.ui.button(label="Yes", style=disnake.ButtonStyle.success)
+    async def success(self, button: disnake.ui.Button, inter: disnake.MessageInteraction):
+
+        # In this example we simply respond to the interaction with a message
+        # then explicitly stop the View from continuing to listen.
+        await inter.response.send_message("Contact us at https://discord.gg/disnake!")
+        self.stop()
+
+    @disnake.ui.button(label="No", style=disnake.ButtonStyle.danger)
+    async def no(self, button: disnake.ui.Button, inter: disnake.MessageInteraction):
+
+        await inter.response.send_message("Got it. Signing off!")
+        self.stop()
+
+
+...
+...
+# The slash command that will respond with a message and the View.
+@bot.slash_command()
+async def buttons_View(inter: disnake.ApplicationCommandInteraction):
+    await inter.response.send_message("Need help?", view=ButtonView())
+```
+
+## Handling View timeout and stopping Views
+
+One of the key advantages of using a <DocsLink reference="disnake.ui.View">View</DocsLink> is the streamlined management
+of your components' state and the convenient setting up and handling of timeouts.
+
+By default, when a View has timed out, the buttons remain attached to the message and appear to be interactable, however, if a user attempts to
+interact with one of thse buttons, an error is displayed. This is also applies `View`s that have been explicitly stopped. To avoid this poor
+user experience, we can customize what happens when a `View` has timed out by altering the <DocsLink reference="disnake.ui.View.on_timeout">on_timeout</DocsLink>
+method and then updating our button callbacks to manage the state of components when the `View` is to be stopped.
+
+Below is an example of handling this type of implementation.
+
+```python title="timeout.py"
+# At top of file
+import disnake
+from disnake.ext import commands
+
+
+class TimeoutButtonView(disnake.ui.View):
```

Comment:
There are too many comments here, imho. Explanations are great (that's the point of this guide, after all :p ), but it might be difficult to keep track of what's happening, when every other line has an entire paragraph of text before it; I hope that makes sense.

Reply:
Yep.  I'd rather have too much info then trim it back, so this is good for me.  :D

Reply:
Fair enough.
The comments in the `yes` and `no` methods here are completely identical, so those should at least be trimmed a bit, explaining it once should be enough.

Reply:
Reworded the comments to make it just a tad shorter, and removed duplicate